### PR TITLE
QUIC: Assign thread before call QUICNetVConnection::start

### PR DESCRIPTION
**Pull request by scw00**

### CODE REVIEW

File: iocore/net/QUICNetProcessor.cc [hunk 1]
```diff
@@ -153,6 +153,11 @@ QUICNetProcessor::connect_re(Continuation *cont, sockaddr const *remote_addr, Ne
   vc->action_     = cont;
 
   SET_CONTINUATION_HANDLER(vc, &QUICNetVConnection::startEvent);
+  if (t->is_event_type(opt->etype)) {
+    vc->thread = t;
+  } else {
+    vc->thread  = eventProcessor.assign_thread(opt->etype);
```
Comment (masaori335) [inline iocore/net/QUICNetProcessor.cc]: It looks like clang-format doesn't like this line.

**Comment by scw00** [comment]
[approve ci ubuntu]

**Comment by maskit** [comment]
@masaori335 clang-format doesn't like your code too :)

**Comment by masaori335** [comment]
@maskit oops, fixed

**Comment by scw00** [comment]
[approve ci ubuntu clang-format]

**Comment by scw00** [comment]
[approve ci ubuntu]

**Comment by scw00** [comment]
[approve ci clang-format]

**Comment by maskit** [comment]
@masaori335 UnixNetProcessor may not use it, but who is responsible for setting `vc->thread' ? Setting it in different places doesn't make sense. Is there a reason we should not do the same thing?

**Comment by masaori335** [comment]
@maskit I meant the bug is QUIC specific and it comes from design of `QUICNetVConnection`. I'm fine with changing `UnixNetProcessor` for consistency.

**Comment by masaori335** [comment]
Let's ship this.

**Pull request closed without merging**